src/server/twitterSearch.js

```diff
@@ -25,16 +26,39 @@ export const TwitAccess = new Twit({
  * @returns {ImmutableTweet}
  */
 const buildTweetFromRaw = (rawTweet) => {
+  const latLong = findLatitudeLongitude(rawTweet);
   return Builders.TweetBuilder()
     .id(rawTweet.id_str)
     .content(rawTweet.text)
     .date(moment(new Date(rawTweet.created_at)).format('YYYY-MM-DD HH:mm:ss'))
-    .likes(rawTweet.favourites_count || 0)
-    .retweets(rawTweet.retweet_count || 0)
+    .likes(rawTweet.favorite_count)
+    .retweets(rawTweet.retweet_count)
+    .latitude(latLong[0])

```

Comment:
I'd prefer us to return an object with specific keys than an array really. Your call here